Overhaul of existing game scripts

ShadowVolt opened the pull request:
Since I had some free time, I ended up doing some in-depth processing of the existing game scripts.

The amount of changes are too many to list, but in every case I managed to cut down on the line count by several thousand without losing anything important. SoS mineral town ended up getting the most manual changes, since many lines were split in bad locations. I also managed to cleanly remove the bracketed furigana from it, which should allow for better matching. The only one I didn't touch was Mystery dungeon DX, as it's script was a hot mess. The most destructive edits were done to VK4's script; I think that I improved it overall, but it's possible I introduced new issues with it.

I definitely plan to add more game scripts in the future, as it is easier than I had anticipated to extract them. (well, usually.)

(Disclaimer: None of these have been tested in-game, but they all follow the same principles that I used when working on Automata's script, which worked beautifully.)

ShadowVolt committed:
text cleanup of all older scripts + mineral town overhaul

too many changes to list them all. mostly usability changes, along with optimization of lines.

ShadowVolt committed:
readme punctuation change

mathewthe2 commented:
I only checked some of the changes you've made. It seems you've removed the character names from the Persona 4 script. I was thinking whether I should remove them but keep another copy with character names. To be fair, keeping the character names worsen the matching result.

ShadowVolt commented:
Ah yeah, forgot that I did that. From my experience using the script matcher, the less prefixed text is present, the more accurate the match tends to be. (likely due to the search matching large amounts similar lines based off that prefix) Though I haven't played P4 in Japanese, so I'm not sure if that messes with the OCR part. (like the way lines are structured visually in-game)

---
Edits:
Another big change that I forgot to mention was that in CrossCode I replaced the tag-style references to items/locations with just placeholder brackets, which could impact things on the user-side. They were formatted as variable names of sorts in English, rather than the Japanese names. (I'll probably test this one later, since I have the game installed.)

I also removed all the whitespace from Animal Crossing. For some reason almost every segment of text was padded with spaces, and there were control codes mixed in there too; so I elected to just remove all of it.
